rules/sources/biometric_data.yaml

```diff
@@ -1,13 +1,13 @@
 sources:
-  - id: Data.Sensitive.BiometricData.FingerprintScans
-    name: Fingerprint Scans
-    category: Biometric Data
-    isSensitive: True
-    sensitivity: high
-    patterns:
-      - "(?i)finger[^\\s/(;)#|,=!>]{0,5}print|finger[^\\s/(;)#|,=!>]{0,5}print[^\\s/(;)#|,=!>]{0,5}scan"
-    tags:
-      law: GDPR
+  # - id: Data.Sensitive.BiometricData.FingerprintScans
```

Comment:
Reason to remove this?

Reply:
We need some enhanced rule to differentiate between biometric fingerprint and browser fingerprints. Till then we decided on removing this, as  the results get incorrectly shown under biometric data.